uiowa/uiowa
# [Emergency site] Rave integration for alerts

Closes #7502 

# How to test
- Code review. I believe I removed all code that would update or delete existing hawk alerts but a second set of eyes would be appreciated.
- `ddev blt ds --site=emergency.uiowa.edu && ddev drush @emergency.local config:import --source ../config/sites/emergency.uiowa.edu --partial -y && ddev drush @emergency.local cim -y && ddev drush @emergency.local cr && ddev drush @emergency.local uli`
- Import command: `ddev drush @emergency.local rave-alerts`
- Output should say 1 alert created, go to the content overview page to see it. Title, body, and Hawk Alert Identifier should match the headline, description, and identifier at the test endpoint - https://content.getrave.com/cap/uiowatest/channel1. Identifier should only appear on edit form and not be editable.
- Change the text of the title and/or body. 
- Run the import command again, output should now say 1 skipped.


Before merging I will change line 70 of EmergencyAPI.php to use BASE_URL_2 instead of BASE_URL_1. You can also make this change locally to test the prod endpoint instead of stage if you'd like.
